client/src/components/navigation/Sidebar.tsx

```diff
@@ -0,0 +1,312 @@
+        <Collapse in={menuItem} timeout="auto" unmountOnExit>
+          <List sx={{ marginLeft: 3, marginTop: -1 }}>
+            {item.children.map((child: NavItemChildProps) =>
+              child.permissions && auth.permissions.length === 0 ? null : (
+                <ListItemButton
+                  component={Link}
+                  to={child.path}
+                  sx={{
+                    marginLeft: 2.1,
+                    marginBottom: 0.5,
+                    minHeight: 22,
+                    borderRadius: 1,
+                    typography: "body2",
+                    color: "text.secondary",
+                    textTransform: "capitalize",
+                    fontWeight: "fontWeightMedium",
+                    ...(child.path === pathname && {
+                      color: "primary.main",
+                      fontWeight: "fontWeightSemiBold",
+                      bgcolor: (theme) => alpha(theme.palette.primary.main, 0.08),
+                      "&:hover": {
+                        bgcolor: (theme) => alpha(theme.palette.primary.main, 0.16),
+                      },
+                    }),
+                  }}
+                >
+                  <Box component="span">{child.title} </Box>
+                </ListItemButton>
+              )
+            )}
+          </List>
+        </Collapse>
+      </>
+    );
+  } else if (item.path !== undefined) {
+    return (
+      <ListItemButton
+        component={Link}
+        to={item.path}
+        sx={{
+          minHeight: 44,
+          borderRadius: 1,
+          typography: "body2",
+          color: "text.secondary",
+          textTransform: "capitalize",
+          fontWeight: "fontWeightMedium",
+          ...(active && {
+            color: "primary.main",
+            fontWeight: "fontWeightSemiBold",
+            bgcolor: (theme) => alpha(theme.palette.primary.main, 0.08),
+            "&:hover": {
+              bgcolor: (theme) => alpha(theme.palette.primary.main, 0.16),
+            },
+          }),
+        }}
+      >
+        <Box component="span" sx={{ width: 24, height: 24, mr: 2 }}>
+          <item.icon />
+        </Box>
+        <Box component="span">{item.title} </Box>
+      </ListItemButton>
+    );
+  } else {
+    return (
+      <ListItemButton
+        sx={{
+          minHeight: 44,
+          borderRadius: 1,
+          typography: "body2",
+          color: "text.secondary",
+          textTransform: "capitalize",
+          fontWeight: "fontWeightMedium",
+        }}
+      >
+        <Box component="span" sx={{ width: 24, height: 24, mr: 2 }}>
+          <item.icon />
+        </Box>
+        <Box component="span">{item.title} </Box>
+      </ListItemButton>
+    );
+  }
+}
+
+/**
+ * The Sidebar-Component displays the sidebar on the left side of the screen
+ * We render the account information and the navigation items
+ */
+export default function Sidebar({ openDrawer, onCloseDrawer, onOpenDrawer }: NavProps) {
```

Comment:
We use the arrow function format for functions (() => {...}).


```suggestion
export default const Sidebar = ({ openDrawer, onCloseDrawer, onOpenDrawer }: NavProps) => {
```